BUILD.gn

```diff
@@ -467,14 +467,7 @@ if (should_generate_symbols) {
     }
   }
 
-  if (is_win) {
-    create_dist_template("create_native_symbols_dist") {
-      output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch-native-symbols.7z"
```

Comment:
Do we need to sync this change with devops repo?

Reply:
> Do we need to sync this change with devops repo?

We already don't use these files in devops repo, so this PR is basically a cleanup.